quipucords/tests/scanner/openshift/test_api.py

```diff
@@ -178,6 +186,137 @@ def test_retrieve_operators(ocp_client: OpenShiftApi):
     assert isinstance(operators[-1], LifecycleOperator)
 
 
+@pytest.mark.vcr(
+    VCRCassettes.OCP_ACM,
+    VCRCassettes.OCP_DISCOVERER_CACHE,
+)
+def test_retrieve_acm_metrics(ocp_client: OpenShiftApi):
+    """Test retrieve acm metrics method."""
+    # OCP instance used on VCR preparation should have ACM operator installed,
+    # with active multiClusterHub instance
+    acm_metrics = ocp_client.retrieve_acm_metrics()
+    assert isinstance(acm_metrics, list)
+
+    # The following assertion is based on the assumption that the provisioned cluster
+    # has not undergone any additional config modifications. When creating the
+    # multiClusterHub instance, it is expected that at least one managed cluster will
+    # be discovered, the hub itself. An associated set of keys is expected.
+    # The "available" key within these metrics is expected to be True, indicating that
+    # the cluster is available and configured properly.
```

Comment:
based on the comment, we can add the following assertion

```python
assert len(acm_metrics) > 2, "This test requires at least 2 clusters managed through ACM"
```

Reply:
assert len(acm_metrics) >= 1, "This test requires at least 1 cluster managed through ACM"
If we don't add any clusters, then the only managed cluster found will be the hub itself. This would be enough for this test to work.

Reply:
it won't be hub + local-cluster(name for hub cluster when found on managed clusters)